### ISSUE TITLE
Failed to parse the JSON string!

### ISSUE THREAD

**Issue by nolan-interactive** [Parallels/vagrant-parallels#242]
Seeing the following exception when running `vagrant up`.

```
$ vagrant up
Bringing machine 'default' up with 'parallels' provider...
Failed to parse the JSON string! Please report this as a bug of Parallels
provider: https://github.com/Parallels/vagrant-parallels/issues
```

The JSON string that appears to be incorrect is:

```
"Require password to":[
\"remove-vm\",
\"clone-vm\"
],
```

**Comment by legal90** [comment]
@nolan-interactive Oh, I'm sorry, seems like it is a bug in Parallels Desktop 11.1.1 and 11.1.2. These double quotes should not be escaped.

For now it could be worked around only by manual disabling password restrictions.

It could be done by these commands:
```
$ prlsrvctl set --require-pwd remove-vm:on
$ prlsrvctl set --require-pwd clone-vm:on
```

Or via GUI:
"Parallels Desktop" -> "Preferences" -> "Security" -> "Require Password to:",  turn off all check-boxes:

![pd_security](https://cloud.githubusercontent.com/assets/4846154/12259048/a1ea760a-b91b-11e5-8555-a4617c122581.png)

**Comment by legal90** [comment]
That issue has been fixed in Parallels Desktop 11.1.3
Please, upgrade: http://kb.parallels.com/111603
